Add how-to on how to control compositor behavior regarding client decoration requests (#3490)
canonical/mir: doc/sphinx/how-to/how-to-control-compositor-responses-to-client-decoration-requests.md

# How to Control Compositor Responses to Client Decoration Requests

> This how-to will show you how to specify the behavior of the compositor
> regarding client window decoration requests.
> 
> In general, clients can ask the compositor to choose for them, or request
> server or client side decorations from the compositor excplicilty. MirAL allows
> you to customize how the server deals with these requests using
> {class}`miral::Decorations`.

Reviewer: ```suggestion
Clients can ask the compositor to use server or client side decorations, or request the compositor to choose for them. {class}`miral::Decorations` allows you to customize how the server deals with these requests.

This how-to will show you how to specify the behavior of the compositor regarding these requests.
```
